deeptime/markov/msm/tram/_tram.py

```diff
@@ -0,0 +1,708 @@
+              matrix (from thermodynamic state k). Two thermodynamic states k and l are defined to overlap at Markov
+              state n if a replica exchange simulation [2]_ restricted to state n would show at least one transition
+              from k to l or one transition from from l to k. The expected number of replica exchanges is estimated from
+              the simulation data. The minimal number required of replica exchanges per Markov state can be increased by
+              decreasing `connectivity_factor`.
+            * 'BAR_variance' : like 'post_hoc_RE' but with a different condition to define the thermodynamic overlap
+              based on the variance of the BAR estimator [3]_. Two thermodynamic states k and l are defined to overlap
+              at Markov state n if the variance of the free energy difference Delta f_{kl} computed with BAR (and
+              restricted to conformations form Markov state n) is less or equal than one. The minimally required variance
+              can be controlled with `connectivity_factor`.
+            * 'summed_count_matrix' : all thermodynamic states are assumed to overlap. The connected set is then
+              computed by summing the count matrices over all thermodynamic states and taking it's largest strongly
+              connected set. Not recommended!
+        maxiter : int, optional, default=10000
+            The maximum number of self-consistent iterations before the estimator exits unsuccessfully.
+        maxerr : float, optional, default=1E-15
+            Convergence criterion based on the maximal free energy change in a self-consistent
+            iteration step.
+        save_convergence_info : int, optional, default=0
+            Every saveConvergenceInfo iteration steps, store the actual increment and the actual log-likelihood; 0
+            means no storage.
+        connectivity_factor : float, optional, default=1.0
+            Only needed if connectivity='post_hoc_RE' or 'BAR_variance'. Values greater than 1.0 weaken the connectivity
+            conditions. For 'post_hoc_RE' this multiplies the number of hypothetically observed transitions. For
+            'BAR_variance' this scales the threshold for the minimal allowed variance of free energy differences.
+
+        References
+        ----------
+        .. [1] Wu, H. et al 2016
+            Multiensemble Markov models of molecular thermodynamics and kinetics
+            Proc. Natl. Acad. Sci. USA 113 E3221--E3230
+        .. [2]_ Hukushima et al, Exchange Monte Carlo method and application to spin
+            glass simulations, J. Phys. Soc. Jan. 65, 1604 (1996)
+        .. [3]_ Shirts and Chodera, Statistically optimal analysis of samples
+            from multiple equilibrium states, J. Chem. Phys. 129, 124105 (2008)
+        """
+        super(TRAM, self).__init__()
+
+        self.lagtime = lagtime
+        assert count_mode == 'sliding', 'Currently the only implemented count_mode is \'sliding\''
```

Comment:
do we even want a `count_mode` parameter then?

Reply:
Oops, we do. That assert is a relic from pyemma. The count modes are all the ones available in the TransitionCountEstimator.